modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -0,0 +1,73 @@
+= Using MUnit Coverage in Studio
```

Comment:
Is there a way to make this more user-task-focused? Like "Report on xxx Coverage Using MUnit"?

Reply:
The repost is a feature that enhances API testing. Let’s see if my rewording works.
Thanks!